src/codeflare_sdk/common/widgets/widgets.py

```diff
@@ -171,15 +146,12 @@ def _on_list_jobs_button_click(self, b):
         from codeflare_sdk import Cluster
 
         cluster_name = self.classification_widget.value
-        namespace = self.ray_clusters_df[
-            self.ray_clusters_df["Name"] == self.classification_widget.value
-        ]["Namespace"].values[0]
 
         # Suppress from Cluster Object initialisation widgets and outputs
         with widgets.Output(), contextlib.redirect_stdout(
             io.StringIO()
         ), contextlib.redirect_stderr(io.StringIO()):
-            cluster = Cluster(ClusterConfiguration(cluster_name, namespace))
+            cluster = Cluster(ClusterConfiguration(cluster_name, self.namespace))
```

Comment:
I have a small concern on this one iirc we have it set where you get a warning if you are using an unsupported Python Version so imagine if someone is using Py3.12 with their own 3.12 image there could potentially be some confusion for them. 

Even though we don't actively encourage users to use unsupported versions it still could be possible that they do for whatever their reasons. 
We could potentially bypass this warning all together by specifying a random string for the image. (It shouldn't matter here at all because they will never interact with this ClusterConfiguration) 

WDYT?